Emby.Dlna/PlayTo/PlayToManager.cs

```diff
@@ -90,10 +90,10 @@ private async void OnDeviceDiscoveryDeviceDiscovered(object sender, GenericEvent
             string location = info.Location.ToString();
 
             // It has to report that it's a media renderer
-            if (usn.IndexOf("MediaRenderer:", StringComparison.OrdinalIgnoreCase) == -1 &&
-                nt.IndexOf("MediaRenderer:", StringComparison.OrdinalIgnoreCase) == -1)
+            if (!usn.Contains("MediaRenderer:", StringComparison.OrdinalIgnoreCase)
+                && !nt.Contains("MediaRenderer:", StringComparison.OrdinalIgnoreCase))
```

Comment:
I thought we were moving from `Contains` to `IndexOf`?

Reply:
No, why would we? The reason this used to be `IndexOf` is that `Contains` didn't have a `StringComparison` overload until netstandart2.1 or something